catalog/app/containers/Bucket/File.js

```diff
@@ -191,7 +191,7 @@ function Analytics({ analyticsBucket, bucket, path }) {
   const formatDate = (date) =>
     dateFns.format(
       date,
-      today.getFullYear() === date.getFullYear() ? `D MMM` : `D MMM YYYY`,
+      today.getFullYear() === date.getFullYear() ? `d MMM` : `d MMM yyyy`,
```

Comment:
do we really need a template string here?

Reply:
nope. good catch. i wonder why prettier didnt catch that

Reply:
```suggestion
      today.getFullYear() === date.getFullYear() ? 'd MMM' : 'd MMM yyyy',
```